spec/features/ingredients/index_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "rails_helper"
+
+RSpec.describe "ingredients index page", type: :feature do
+  it "shows a list of all ingredients with their attributes, sorted alphabetically" do
+    cheese = Ingredient.create!(name: "Cheese", cost: 3)
+    noods = Ingredient.create!(name: "Noodles", cost: 2)
+    herbals = Ingredient.create!(name: "Herbs", cost: 1)
+
+    visit "/ingredients"
+
+    expect(current_path).to eq("/ingredients")
+    expect(page).to have_content(cheese.name)
+    expect(page).to have_content(noods.name)
+    expect(page).to have_content(herbals.name)
+    expect(page).to have_content(cheese.cost)
+    expect(page).to have_content(noods.cost)
+    expect(page).to have_content(herbals.cost)
+    expect(page).to have_content("Cheese: 3")
```

Comment:
You can combine these two different types of assertions by doing:
`expect(page).to have_content("#{cheese.name}: #{cheese.cost}")`